docker/server/config/config-mysql-grpc.properties

```diff
@@ -13,6 +13,9 @@ db=mysql
 
 jdbc.url=jdbc:mysql://mysql:3306/conductor
 
+conductor.mysql.connection.pool.size.max=10
```

Comment:
Nice Work @davidwadden, thanks. Is there a reason to set mysql parameters explicitly? If so, and if it is configurable, shall we add comments mentioning the same.

Reply:
@kishorebanala On `master`, building the gRPC docker images under [`/docker/grpc`](https://github.com/Netflix/conductor/tree/master/docker/grpc) results in `conductor-server` aborting startup with the following logs (also reproduced outside of Docker):

https://gist.github.com/davidwadden/6590bf56ff01a0aca2ec7639fd63dab7

Looking at [`MySQLConfiguration.java`](https://github.com/Netflix/conductor/blob/v2.1.0/mysql-persistence/src/main/java/com/netflix/conductor/mysql/MySQLConfiguration.java#L25-L30), which to the commenter's point [are the `HikariConfig` defaults](https://github.com/brettwooldridge/HikariCP/blob/HikariCP-3.2.0/src/main/java/com/zaxxer/hikari/HikariConfig.java#L111-L112), the default values of `-1` prevent startup on Conductor.

The values of `2` for `minIdle` and `10` for `maxPoolSize` were arbitrary values.

Reply:
Great, would you mind adding the same in comments please. It was not clear as to why these values are chosen in first look.